src/test/v2.0/support/channels-support-2.0.ts

```diff
@@ -10,41 +10,22 @@ declare let fdc3: DesktopAgent;
 export class ChannelControl2_0 implements ChannelControl<Channel, Context, Listener> {
   private readonly testAppChannelName = "test-channel";
 
-  retrieveAndJoinChannel = async (channelNumber: number, channelId?: string): Promise<Channel> => {
-    if (channelNumber) {
-      const channel = await this.getUserChannel(channelNumber);
-      await fdc3.joinUserChannel(channel.id);
-      return channel;
-    } else if (channelId) {
-      const channel = await this.getUserChannel(undefined, channelId);
-      await fdc3.joinUserChannel(channelId);
-      return channel;
-    } else {
-      throw new Error("The retrieveAndJoinChannel function requires at least one parameter to be passed to it");
-    }
-  };
-
-  getSystemChannels = async () => {
-    return await fdc3.getUserChannels();
-  };
-
-  leaveChannel = async () => {
-    return await fdc3.leaveCurrentChannel();
-  };
+  getNonGlobalUserChannels = async () => {
+    const channels = await fdc3.getUserChannels();
+    return channels.filter(channel => channel.id.indexOf('global') === -1);
+  }
 
-  getUserChannel = async (channelNumber: number, channelId?: string): Promise<Channel> => {
+  getNonGlobalUserChannel = async (): Promise<Channel> => {
     const channels = await fdc3.getUserChannels();
```

Comment:
can we get this to call getNonGlobalUserChannels() and return the first element of it